skopt/optimizer/gp.py

```diff
@@ -206,14 +207,15 @@ def gp_minimize(func, dimensions, base_estimator=None,
     # Check params
     rng = check_random_state(random_state)
     space = normalize_dimensions(dimensions)
-    base_estimator = cook_estimator("GP", space=space, random_state=rng,
-                                    noise=noise)
+    base_estimator = cook_estimator(
+        "GP", space=space, random_state=rng.randint(0, np.iinfo(np.int32).max),
```

Comment:
I am wondering should we introduce a function in `skopt.utils` that does sampling of random integer based on random state variable? Might be less verbose then.

Reply:
I'm -0 on that. We could add a `maxrange_randint()` or so but it would only be useful for the special case where you want the max range.

Reply:
Yeah then lets leave it as it is.